src/database/DataTypes/Item.js

```diff
@@ -9,7 +9,8 @@ export class Item extends Realm.Object {
   }
 
   get totalQuantity() {
-    return getTotal(this.batches, 'totalQuantity');
+    const item = this.crossReferenceItem ? this.crossReferenceItem : this;
```

Comment:
we do this check quite often, i think we should make and use a getter item.realItem => this.crossReferenceItem || this.